### PR TITLE
VACMS-7068 claro login page

### PR DESCRIPTION
**Pull request by tjheffner**
## Description

Relates to #7068. Also includes a fix for vagovclaro to work with the updated helptext fix. :)

This is another iteration on the new login experience... there will likely be another iteration after design/UX have a chance to look at it. 

Code in this PR located in `vagovclaro.theme` could probably move to the `va_gov_help_center` or `va_gov_login` modules when this theme is turned on, but I kept it in the theme for now to prevent changing the current experience... happy to make those updates there now (PIV / Get help) if that is appropriate.

## Testing done
Flipped `activate: true` in `config/local/simplesamlphp_auth.settings.yml` in order to see the PIV login locally. Otherwise, looked at the login page with site alert turned on and off.

## Screenshots
Screen Shot 2021-12-15 at 1 33 43 PM

## QA steps
1. enable vagovclaro as the default theme
2. log out to see the new login experience
3. ???

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [x] Acceptance Criteria in related issue are met.
- [x] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS team`
- [x] `Sitewide CMS Team`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [x] No announcement is needed for this code change.
  - [x] Merge & carry on unburdened by announcements

No announcement necessary until the turn-on date is closer at hand.

### CODE REVIEW

File: docroot/themes/custom/vagovclaro/templates/form/form-element--textarea.html.twig [hunk 1]
```diff
@@ -0,0 +1,96 @@
+{#
```
Comment (tjheffner) [inline docroot/themes/custom/vagovclaro/templates/form/form-element--textarea.html.twig:1]: noting this file was added to vagovadmin in a [recent PR](https://github.com/department-of-veterans-affairs/va.gov-cms/commit/87fee2bc9fc3b9090603b9398532cef82809d289), but not to vagovclaro, which broke some node forms. has no functionality changes for the purposes of this PR.

File: docroot/themes/custom/vagovclaro/vagovclaro.theme [hunk 2]
```diff
@@ -5,12 +5,19 @@
...
 /**
  * Implements hook_preprocess_html().
  */
 function vagovclaro_preprocess_html(&$variables) {
   $is_admin = \Drupal::service('va_gov_user.user_perms')->hasAdminRole(TRUE) ? 'admin' : 'not-admin';
   $variables['attributes']['class'][] = 'role-' . $is_admin;
+
+  // For css that is different per environment (admin toolbar color, etc.)
+  $environment_name = getenv('CMS_ENVIRONMENT_TYPE');
+  $variables['attributes']['class'][] = 'cms-' . $environment_name;
```
Comment (tjheffner) [inline docroot/themes/custom/vagovclaro/vagovclaro.theme:20]: this bit isn't used yet but could be a handy helper for targeting environment specific css in the future (admin toolbar colors, etc)

File: docroot/themes/custom/vagovclaro/templates/page/page--user--login.html.twig [hunk 3]
```diff
@@ -57,7 +61,9 @@
         {{ page.help }}
       </div>
     {% endif %}
+
+    <p>A mission statement for the CMS editors could go in this space.</p>
```
Comment (swirtSJW) [inline docroot/themes/custom/vagovclaro/templates/page/page--user--login.html.twig:65]: Recommend: To boldly go where we've already lost a few dozen people before.  ;)

**Review by swirtSJW — APPROVED** [review]
Nice work.  Definitely appreciate the clarifying comments you leave so I don't have to figure out mysteries.

**Pull request merged**